### PR TITLE
`callback_url` should be redefined to let namecheap use google analytics tags in their params

### PR DESCRIPTION
**Pull request by sophiedeziel**
Hi Rachel!

### CODE REVIEW

File: lib/omniauth/strategies/namecheap.rb [hunk 1]
```diff
@@ -10,6 +10,10 @@ class Namecheap < OmniAuth::Strategies::OAuth2
                               token_url:     '/apps/sso/api/token',
                               me_url:        '/apps/sso/api/resource/user'
 
+      def callback_url
+        full_host + script_name + callback_path
```
Comment (rachel-carvalho) [inline lib/omniauth/strategies/namecheap.rb:14]: Looking at the source for [this method](https://github.com/omniauth/omniauth/blob/master/lib/omniauth/strategy.rb#L439), it seems that we're removing the query string here. Could you add a test that shows it?

**Review by rachel-carvalho — CHANGES_REQUESTED** [review]
Hey Sophie :)

Sorry I took so long to check your PR, I was on vacation 🎉

**Comment by rachel-carvalho** [comment]
I'll close this PR because I realized that it wasn't the `callback_url` that was wrong, but the `redirect_uri` that we were passing when trying to retrieve the token. It didn't match the original one we passed on the request phase and that's why it was being denied.

**Comment by sophiedeziel** [comment]
Yay!

**Pull request closed without merging**